config.py

```diff
@@ -786,6 +786,8 @@ def create_transaction(db):
     FEATURE_QUOTA_MANAGEMENT = False
     # default value for all organizations to reject by default. 0 = no configuration
     DEFAULT_SYSTEM_REJECT_QUOTA_BYTES = 0
+    # Time delay for quota backfill
```

Comment:
Can you add some details of why this delay is necessary, and what are the implications.

Reply:
done